doc/advanced_usage.md

```diff
@@ -92,4 +92,5 @@ the status would be 400.
 
 ## Spectroscopic lines
 
-The line list implemented on SkyPortal was derived using the [NIST atomic database](https://www.nist.gov/pml/atomic-spectra-database), employing the methods of [Gal-Yam  2019](https://ui.adsabs.harvard.edu/abs/2019ApJ...882..102G/abstract).
+The selection of element and galaxy lines on SkyPortal were derived from the NIST atomic database (<https://www.nist.gov/pml/atomic-spectra-database>) , using the methods of Gal-Yam (2019) (<https://ui.adsabs.harvard.edu/abs/2019ApJ...882..102G/abstract>). Telluric and Skylines were migrated from the ZTF Marshal.
```

Comment:
Thanks for the PR, @rjoebruch.

We need to be a bit more specific here.  SkyPortal is for a wider audience than ZTF, so many of our users have no way of understanding what "migrated from the ZTF Marshal means".

Reply:
At some point we might have to admit that we are using these lines because we've always been using these lines and no one knows anything deeper than that. 
But I can try to ask around if anyone has a better reference.

Reply:
That would be a sad state of affairs, but if so we should say it.

Reply:
@stefanv what do you say?

Reply:
To my uninformed eye, that just looks like a random website with data files on it. Do we really have no sense of how these came about?

Reply:
I agree the specific website is not very informative. I think the skylines and tellurics are just "well known". I don't know if this is really going to matter to anyone, I suggest we leave the text as-is.

Reply:
@profjsb maybe you can weigh in on this issue?

Reply:
I'm fine with the wording as is.